packages/app/src/utils/styledShouldForwardProp.ts

```diff
@@ -0,0 +1,5 @@
+import shouldForwardProp from '@styled-system/should-forward-prop';
+
+type StyledShouldForwardPropType = (prop: string | number) => boolean;
+
+export const StyledShouldForwardProp = shouldForwardProp as StyledShouldForwardPropType;
```

Comment:
What you are exporting is a (typed) value, not a type itself.

```suggestion
export const styledShouldForwardProp = shouldForwardProp as StyledShouldForwardPropType;
```